package-lock.json

```diff
@@ -0,0 +1,6253 @@
+{
+  "name": "call_of_duty",
+  "version": "1.0.0",
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {
+    "": {
+      "name": "call_of_duty",
+      "version": "1.0.0",
+      "license": "ISC",
+      "dependencies": {
+        "fastify": "^4.11.0",
+        "vitest": "^0.26.3"
```

Comment:
You remove vitest form dependencies by deleting it - this is not good because it doesn't change the `package-lock`.
You should not change the packages without command.
 To move from prod to dev: `npm i <module_name> -D`
 To move from dev to prod: `npm i <module_name> -P`
 To remove: `npm uninstall <module_name>`

Reply:
done